src/components/BountyCard/index.js

```diff
@@ -54,31 +60,43 @@ const BountyCard = props => {
               <FontAwesomeIcon icon={faSeedling} />
             </Text>
           </div>
-          <Text color="black">FAKE Beginner </Text>
+          <Text color="black" style="Body" weight="font-weight-bold">
+            FAKE Beginner{' '}
+          </Text>
           <div className={`${styles.dataCategory}`}>
-            <Text color="grey">Difficulty</Text>
+            <Text color="grey" style="Body">
+              Difficulty
+            </Text>
           </div>
         </div>
         <div className={`${styles.dataCell}`}>
           <div className={`${styles.dataIcon}`}>
-            <Text color="grey">
+            <Text color="grey" style="Body">
               <FontAwesomeIcon icon={faClock} />
             </Text>
           </div>
-          <Text color="black">{moment(deadline).fromNow(true)}</Text>
+          <Text color="black" style="Body" weight="font-weight-bold">
+            {moment(deadline).fromNow(true)}
```

Comment:
@corwinharrell have you looked at/played with all moment's formatters? We can always change the format, etc if need be.

Reply:
@villanuevawill I have not but that's cool.